consensus/enclave/api/src/lib.rs

```diff
@@ -42,12 +42,12 @@ pub struct WellFormedEncryptedTx(pub Vec<u8>);
 /// Tx data we wish to expose to untrusted from well-formed Txs
 #[derive(Clone, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
 pub struct WellFormedTxContext {
```

Comment:
This struct is now ordered by fee, then by tx_hash.

Reply:
Having a comment mentioning that the ordering of the fields in this struct have an effect is probably a good idea.